### PR TITLE
ref: feedbackAlertBanner link to docs

### PR DESCRIPTION
**Pull request by bruno-garcia**
Follow up to:
* https://github.com/getsentry/sentry/pull/78964/

I imagine the linter will be mad at me

### CODE REVIEW

File: static/app/views/alerts/rules/issue/feedbackAlertBanner.tsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ export default function FeedbackAlertBanner({
   return filterFeedback ? (
     <StyledFeedbackAlert showIcon type="info">
       {tct(
-        'This issue category condition is ONLY for feedbacks from the built-in widget. Crash-report feedback alerts can be enabled in [link:Project Settings.]',
+        'This issue category condition is ONLY for feedbacks from the [built-in widget](https://docs.sentry.io/platforms/javascript/user-feedback/#user-feedback-widget). [Crash-report Modal](https://docs.sentry.io/platforms/javascript/user-feedback/#crash-report-modal) alerts can be enabled in [link:Project Settings.]',
```
Comment (ryan953) [inline static/app/views/alerts/rules/issue/feedbackAlertBanner.tsx:24]:
```suggestion
        'This issue category condition is ONLY for feedbacks from the [built-in widget](https://docs.sentry.io/platforms/javascript/user-feedback/#user-feedback-widget). [Crash-report Modal](https://docs.sentry.io/platforms/javascript/user-feedback/#crash-report-modal) alerts can be enabled in [link:Project Settings].',
```

**Review by ryan953 — APPROVED** [review]

**Review by michellewzhang — DISMISSED** [review]
SCR-20241010-minj

vercel preview looks like this, i think we need to insert the links in the `tct`

**Comment by michellewzhang** [comment]
pushed up a change to your branch

**Review by michellewzhang — APPROVED** [review]

File: static/app/views/alerts/rules/issue/feedbackAlertBanner.tsx [hunk 2]
```diff
@@ -21,9 +22,15 @@ export default function FeedbackAlertBanner({
   return filterFeedback ? (
     <StyledFeedbackAlert showIcon type="info">
       {tct(
-        'This issue category condition is ONLY for feedbacks from the built-in widget. Crash-report feedback alerts can be enabled in [link:Project Settings.]',
+        'This issue category condition is ONLY for feedbacks from the [linkWidget:built-in widget]. [linkModal: Crash-report modal] alerts can be enabled in [link:Project Settings.]',
```
Comment (michellewzhang) [inline static/app/views/alerts/rules/issue/feedbackAlertBanner.tsx:25]:
```suggestion
        'This issue category condition is ONLY for feedbacks from the [linkWidget:built-in widget]. [linkModal: Crash-report modal] alerts can be enabled in [link:Project Settings].',
```

File: static/app/views/alerts/rules/issue/feedbackAlertBanner.tsx (hunk 1 above)
Comment (michellewzhang) [reply static/app/views/alerts/rules/issue/feedbackAlertBanner.tsx]: applied this change

**Comment by bruno-garcia** [comment]
woo thank you!

**Pull request merged**